src/components/EditorContent/index.jsx

```diff
@@ -1,23 +1,49 @@
-import React from "react";
+import React, { useEffect, useRef } from "react";
 
 import classnames from "classnames";
 import DOMPurify from "dompurify";
+import { copyToClipboard } from "neetocommons/utils";
+import { Copy } from "neetoicons";
+import { Button } from "neetoui";
+import ReactDOM from "react-dom";
 
 import { EDITOR_CONTENT_CLASSNAME, SANITIZE_OPTIONS } from "./constants";
 import { highlightCode, substituteVariables } from "./utils";
 
+const CopyButton = ({ onClick }) => (
+  <Button icon={Copy} size="small" style="secondary" onClick={onClick} />
+);
+
 const EditorContent = ({
   content = "",
   variables = [],
   className,
   ...otherProps
 }) => {
+  const editorContentRef = useRef(null);
   const htmlContent = substituteVariables(highlightCode(content), variables);
   const sanitize = DOMPurify.sanitize;
 
+  useEffect(() => {
+    const preTags = editorContentRef.current?.querySelectorAll(
+      ".neeto-editor-content pre"
+    );
+
+    preTags.forEach(preTag => {
+      const button = document.createElement("div");
+      button.className = "copy-button";
+      const handleButtonClick = () => {
+        copyToClipboard(preTag.textContent);
```

Comment:
Also check if `Toastr.info` works here. To test this out, you might need to add the `ToastContainer` as well. Else, directly test this out in an host app.

Reply:
@AbhayVAshokan This will also trigger a Toastr.